src/state/tracks/tracksActionCreator.js

```diff
@@ -0,0 +1,24 @@
+import * as types from './tracksActionTypes';
+import codeClanApi from '../../api/apiUtils';
+import history from '../../history';
```

Comment:
Codacy found an issue: ['history' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5801772)